README.rst

```diff
@@ -76,6 +76,10 @@ What's New?
 in development
 ^^^^^^^^^^^^^^
 
+pydoctor 22.5.1
+^^^^^^^^^^^^^^^
+* Pin ``docutils>=0.17`` to avoid crashing with ``AttributeError`` when processing type fields.
+
```

Comment:
By "pinning" I understand requiring a very specific version.

This is just requiring a minimum version.

```suggestion
* ``docutils>=0.17`` is now the minimum supported version. This was done to fix crashing with ``AttributeError`` when processing type fields.

```